clay/amphora-html
# Add the algolia search configuration

# Steps for testing the site locally:
- Go to the `website` folder
- Install package using `npm install`
- Start your local server using `npm start`
- Go to [localhost](http://localhost:3000/)
- Check on the top right corner of the page and will be a search input
- Compare with the current [docs page](https://docs.clayplatform.com/amphora-html/)

## Intallation process:
https://docusaurus.io/docs/en/search

## Official Algolia site
https://community.algolia.com/docsearch/

Note:
- We need the `Angolia` api key to create the secret and the `indexName`
